SPECS/xha.spec

```diff
@@ -1,7 +1,7 @@
 Summary: xha - XenServer proprietary HA daemon
 Name:    xha
 Version: 10.1.0
-Release: 2%{?dist}
+Release: 3%{?dist}
```

Comment:
The first digit for the release does not belong to us, it's from upstream. So we don't touch it and add ours behind: `2.1`